services/db/recovery.go

```diff
@@ -58,11 +59,24 @@ func getRecoveryData() RecoveryDataT {
 			panic(err)
 		}
 	}
-
 	var recoveryData RecoveryDataT
 	err = json.Unmarshal(data, &recoveryData)
-	if err != nil {
-		panic(err)
+
+	if err != nil{
+		const timeLayout = "20060102150405"
+		filePath := strings.Split(storagePath,".")
+		newFilePath := fmt.Sprintf("%s-%s.%s",filePath[0],time.Now().Format(timeLayout),filePath[1])
```

Comment:
Instead of the following code, can we make it simple?

```
	err = json.Unmarshal(data, &recoveryData)
    if err != nil {
        pkgLogger.Error(...)
        err = os.Rename(storagePath, fmt.Sprintf("%s.bkp", storagePath)
        if err != nil {
            pkgLogger.Error(...)
        }
        recoveryData = RecoveryDataT{Mode: normalMode}
    }
```

Reply:
Let's just have one backup file - instead of timestamp based. 
If we keep adding files, we will also need to start cleaning up the old files. I think one backup is sufficient.

Reply:
done. converted to single backupFile logic.